test/parallel/test-net-bytes-read.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const net = require('net');
+
+const big = new Buffer(1024 * 1024);
```

Comment:
Understood. We're going to have that problem in general anyway. Best to use the new APIs moving forward tho